buildroot-external/configs/generic_aarch64_defconfig

```diff
@@ -3,7 +3,6 @@ BR2_TOOLCHAIN_BUILDROOT_CXX=y
 BR2_DL_DIR="/cache/dl"
 BR2_CCACHE=y
 BR2_CCACHE_DIR="/cache/cc"
-BR2_OPTIMIZE_2=y
```

Comment:
I see this is the new default, hence config is no longer needed :+1:

Reply:
Yes, basically all the changes here shouldn't change anything, they just fell out of defconfig because they became dependencies or defaults.

Reply:
> Generally LGTM. There are some package checks failing because upstream tags are missing. From my point of view we can also disable that particular check, if possible.

Okay, let's do that for now, and I'll make a note to re-enable it later - it might be actually useful for us too.